aea/cli/utils/config.py

```diff
@@ -160,6 +165,24 @@ def set_cli_author(click_context: click.Context) -> None:
     click_context.obj.set_config("cli_author", cli_author)
 
 
+def get_registry_path_from_cli_config() -> str:
+    """Get registry path from config."""
+    config = get_or_create_cli_config()
+    registry_path = config.get(REGISTRY_PATH_KEY, None)
+    if registry_path:
+        return registry_path
+    registry_path = Path(DEFAULT_REGISTRY_NAME)
+    if registry_path.absolute().is_dir():
+        return str(registry_path)
+    registry_path = Path("..", DEFAULT_REGISTRY_NAME)
+    if registry_path.absolute().is_dir():
+        return str(registry_path)
+    cli_logger.debug(
+        f"Registry path not provided and `packages` not found in current ({os.getcwd()}) and parent directory."
+    )
+    return ""
```

Comment:
I am not happy about the default. I'd prefer to raise an exception, but that is not ideal either as then commands unrelated to the registry stop working for no reason. So it's probably better to raise at the point of using the path. In `Context` getter of `registr_path` we could raise when the path is the empty string.

Reply:
Another question is when and where to set the registry path on cli config. One way to do it is during `aea init`. Otherwise we need another cli command which I would like to avoid..